### PR TITLE
Bug fixes

### LINKED ISSUE #184: IE Promise polyfil
People are using IE... So promises/fetch won't work

### LINKED ISSUE #185: Results eval screen looks jacked on safari
Not sure why
screen shot 2016-09-05 at 2 33 31 pm

### LINKED ISSUE #186: error: null value in column "secondcomments" violates not-null constraint
There shouldn't be a not null constraint...

### LINKED ISSUE #188: Results.approve
Result.checkAllDoneForPerformance { error: invalid input syntax for integer: "{}"

### PR DESCRIPTION
**Pull request by atareshawty**
Fixes #184 
Fixes #186 
Fixes #185 
Fixes #188

### CODE REVIEW

**Comment by btmills** [comment]
Flexbox fixes weren't half as bad as they could have been

**Comment by atareshawty** [comment]
screen shot 2016-09-05 at 4 20 14 pm

File: db/queries/results-for-approval.sql [hunk 1]
```diff
@@ -24,7 +24,7 @@ FROM (
   FROM performances AS p, results AS r1, users AS u1
   WHERE r1.firstnamenumber = u1.namenumber AND r1.performanceid = p.id AND r1.needsapproval AND r1.pending
 ) user1
-JOIN (
+LEFT OUTER JOIN (
```
Comment (atareshawty) [inline db/queries/results-for-approval.sql]: This change gets results for challenges involving one person

File: views/layouts/main.handlebars [hunk 2]
```diff
@@ -4,8 +4,10 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="initial-scale=1, maximum-scale=1" />
+    <meta name="apple-mobile-web-app-capable" content="yes">
     <title>OSUMB Challenges</title>
     <link rel="stylesheet" type="text/css" href="/dist/style.css">
+    <script>(function(a,b,c){if(c in b&&b[c]){var d,e=a.location,f=/^(a|html)$/i;a.addEventListener("click",function(a){d=a.target;while(!f.test(d.nodeName))d=d.parentNode;"href"in d&&(chref=d.href).replace(e.href,"").indexOf("#")&&(!/^[a-z\+\.\-]+:/i.test(chref)||chref.indexOf(e.protocol+"//"+e.host)===0)&&(a.preventDefault(),e.href=d.href)},!1)}})(document,window.navigator,"standalone");</script>
```
Comment (atareshawty) [inline views/layouts/main.handlebars]: Explanation [here](https://gist.github.com/irae/1042167) - Thanks @btmills

**Comment by atareshawty** [comment]
Did a deploy on prod. All of these changes are good to go 🎉

**Pull request merged**